app/v2/services/RetrieveForeignPropertyAnnualSubmissionService.scala

```diff
@@ -51,8 +51,8 @@ class RetrieveForeignPropertyAnnualSubmissionService @Inject()(connector: Retrie
     result.value
   }
 
-  private def ifsErrorMap =
-    Map(
+  private val ifsErrorMap = {
```

Comment:
Do we want to rename `ifsErrorMap` to `downstreamErrorMap` or `errorMap` just to keep it general?